### ISSUE TITLE
Hello，I want to ask when the question about the location of hintText can be fixed

### ISSUE THREAD

**Issue by lucky1213** [memspace/zefyr#293]
## Steps to Reproduce

1. ...
2. ...
3. ...

## Logs

```
```

```
```

**Comment by cgestes** [comment]
nobody is really maintaining the project, at the moment. I made a branch with bug fixes that I slowly stack up :)

So you may have to do the same :)

This issue is already tracked here: https://github.com/memspace/zefyr/issues/163

**Comment by lucky1213** [comment]
thanks

**Comment by lucky1213** [comment]
When running on macos, the deletion and return of real keyboard typing are invalid

**Comment by cgestes** [comment]
The return issue may be fixed by this: https://github.com/cgestes/zefyr/commit/e134d7e084395bf9833dad1e611b5305fa28686f .
Deletion: never saw that issue. (I dont have a mac)

**Comment by namchuai** [comment]
> nobody is really maintaining the project, at the moment. I made a branch with bug fixes that I slowly stack up :)

:(

**Comment by amantoux** [comment]
Is this project still alive? or should we create a fork?

**Comment by cgestes** [comment]
not really alive :/

I maintain my own branch, and try to answer to the issues here when I suspect I can help. 

The code doesn't compile against flutter anymore without patches.

not sure we need a fork, we may just ask the maintainer to give us some access.

I created this issue for a release: https://github.com/memspace/zefyr/issues/289 It didnt have a lot of activity so I suspected the situation was not so bad as it is.

**Comment by amantoux** [comment]
:(
it is promising project

**Comment by cgestes** [comment]
Let's maintain a branch together?

I already have the following:
- compilation for current flutter versions
- some linting
- upgrade the example
- fix return not always working
- remove all crashes when targeting the web
- fix some selection bugs
- support being expandable

I am working on embedding any types of blocs in Zefyr (allowing to embed Zefyr itself). It works but it's not clean yet.
I also have a range of half baked features: 
- support for clicking links
- highlight of text
- mentions

Am I still in the middle of a battle to have a fully working Zefyr on the web. (partly blocked by some Flutter engine features not fully implemented for the web)

**Comment by amantoux** [comment]
Sure!

I’ve just started looking into Zefyr
I’m currently struggling with the SelectionHandlers that don’t behave as in
TextField from a UX perspective.
still trying to find out how to achieve it

Le ven. 15 mai 2020 à 19:45, Cedric GESTES <notifications@github.com> a
écrit :

**Comment by amantoux** [comment]
I've rebased on your master but there still seems to be compiation issues

- main.dart in example : FakeFocusIntent not calling super(this.key)
- input.dart: AutofillScope refers to no class

Also, could you point me to your linter? mine complains in each and every file

**Comment by cgestes** [comment]
Hi Alan,

I'll check the examples. Linter we should define the rules we want to
apply. Its not clean here either.

**Comment by edwin-alvarez** [comment]
when will you be able to upload this fix?

**Comment by pulyaevskiy** [comment]
@edwin-alvarez If your question is about hintText issue then there is still no solution to that. 

I did research the root cause and it turns out that solution would not be trivial, for such a minor issue. It will require a substantial rework of rendering layer in Zefyr. Which I’m planning to do.

**Comment by edwin-alvarez** [comment]
is there a way just to don't show it up. Can that be set to empty as a temporal workaround

**Comment by edwin-alvarez** [comment]
@cgestes is there a way just to don't show it up. Can that be set to empty as a temporal workaround?

**Comment by cgestes** [comment]
Hi,

I dont know, but it should not be hard to make the modification.

**Comment by tangjingyuan** [comment]
I used a trick way to fix it as a temporal workaround by put the ZefyrField inside Stack and put a TextFiled under ZefyrField. Auto hide or show the TextField by listen the editor changes.

See the code:
```
class _EditorRouteState extends State<EditorRoute> {
  ZefyrController _controller;
  FocusNode _focusNode;
  bool hideHint = false;

  @override
  void initState() {
    super.initState();
    _controller = ZefyrController(NotusDocument());
    _controller.addListener(() {
      if (_controller.document.length > 1 && !hideHint) {
        setState(() {
          hideHint = true;
        });
      } else if (_controller.document.length <= 1 && hideHint) {
        setState(() {
          hideHint = false;
        });
      }
    });
    _focusNode = FocusNode();
    _focusNode.requestFocus();
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: Text("Post a New Topic")),
      body: ZefyrTheme(
        data: ZefyrThemeData(
            toolbarTheme: ToolbarTheme.fallback(context).copyWith(
                color: Colors.white,
                iconColor: Colors.black,
               toggleColor: Colors.grey)),
        child: ZefyrScaffold(
          child: ListView(
            children: <Widget>[
              Container(
                  height: 50,
                  child: TextField(
                    decoration: InputDecoration(
                        hintText: "Subject",
                        contentPadding: EdgeInsets.only(left: 16)),
                  )),
              Stack(
                children: <Widget>[
                  hideHint
                      ? Container(height: 0)
                      : Positioned(
                          top: 0,
                          left: 16,
                          child: Container(
                            width: MediaQuery.of(context).size.width,
                            height: 50,
                            child: TextField(
                              decoration: InputDecoration(
                                  border: InputBorder.none,
                                  hintText: "Type here to say something..."),
                            ),
                          ),
                        ),
                  ZefyrField(
                    height: 250,
                    decoration: InputDecoration(
                      contentPadding:
                          EdgeInsets.only(left: 16, right: 16, bottom: 30),
                      border: InputBorder.none,
                      // hintText: "Type Here to say something...",
                    ),
                    // padding: EdgeInsets.all(16),
                    controller: _controller,
                    focusNode: _focusNode,
                    imageDelegate: _ImageDelegate(),
                  ),
                ],
              ),
            ],
          ),
        ),
      ),
    );
  }
}
```

And the screenrecord:
![21595411530_ pic_hd](https://user-images.githubusercontent.com/12523498/88162601-1d525980-cc44-11ea-9116-e64f17ec1832.jpg)

**Comment by edwin-alvarez** [comment]
@tangjingyuan, what a great WORKAROUND.....
